### PR TITLE
[1LP][RFR]Added new collection for taging api testing

### PR DESCRIPTION
**Pull request by rlbabyuk**
Purpose or Intent
=================
New collections for REST taggign testing
{{pytest: cfme/tests/configure/test_tag.py}}

### CODE REVIEW

File: cfme/tests/configure/test_tag.py [hunk 1]
```diff
@@ -198,9 +198,16 @@ def test_create_tag_with_wrong_arguments(self, appliance):
 
     @pytest.mark.tier(3)
     @pytest.mark.meta(blockers=[BZ(1451025, forced_streams=['5.7'])])
+    @pytest.mark.uncollectif(
+        lambda appliance, collection_name: appliance.version < '5.9' and
+        collection_name in ["availability_zones", "cloud_networks", "cloud_networks",
+                            "cloud_subnets", "flavors", "network_routers", "security_groups"],
+        reason="5.8 version doesn't support all available collection for tagging"
```
Comment (bsquizz) [inline cfme/tests/configure/test_tag.py]: Can you add which collection name we're skipping in the reason so the reason for uncollecting is very clear? For example: 
```python
reason="appliances <5.9 do not support tagging with collection '{}'".format(collection_name)
```

Comment (rlbabyuk) [reply cfme/tests/configure/test_tag.py]: collection_name is not accessible for the reason in uncollectif

Comment (bsquizz) [reply cfme/tests/configure/test_tag.py]: True, just noticed that's coming from the lambda ;)

Comment (psav) [reply cfme/tests/configure/test_tag.py]: We could expose that.....but its probably a black hole

**Pull request merged**